keanu-project/src/main/java/io/improbable/keanu/vertices/intgr/probabilistic/FuzzyCastToIntegerVertex.java

```diff
@@ -12,6 +12,12 @@
 
 import static org.apache.commons.math3.special.Erf.erf;
 
+/**
+ * Takes a double and casts it to an integer with a user definable level of fuzziness over the value cast to. The range
+ * of potential integer values cast to is specified with a min and max (inclusive). The probability of casting to a
+ * given integer is represented as a Gaussian distribution centred on the input value, with a use specifiable sigma.
+ * E.n., a sigma value of 0 will guarantee casting ot the nearest integer value with half up rounding.
```

Comment:
ot -> to

Reply:
Is E.n. a common abbreviation? I haven't seen it before. Assume it means end note?